bindu/auth/hydra/registration.py

```
@@ -271,32 +179,19 @@ async def register_agent_in_hydra(
-                    vault_stored = await vault_client.store_hydra_credentials(
-                        credentials
-                    )
-                    if vault_stored:
-                        logger.info("✅ Hydra credentials backed up to Vault")
-                    else:
-                        logger.warning("⚠️  Failed to backup Hydra credentials to Vault")
-                except Exception as e:
-                    logger.warning(f"Failed to backup credentials to Vault: {e}")
+                    await vault_client.store_hydra_credentials(credentials)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)